foreign_cc/built_tools/private/built_tools_framework.bzl

```diff
@@ -44,12 +54,18 @@ def built_tool_rule_impl(ctx, script_lines, out_dir, mnemonic):
     """
 
     root = detect_root(ctx.attr.srcs)
+    lib_name = ctx.attr.name
+    env_prelude = get_env_prelude(ctx, lib_name, [], "")
 
-    script = [
+    cc_toolchain = find_cpp_toolchain(ctx)
+    linker_path_str = "\"" + paths.dirname(cc_toolchain.ld_executable).replace("C:", "/c") + "\""
```

Comment:
What if the build is executed from a drive other than C?

Reply:
Would it be possible to use a relative path? If the env prelude from the common framework is being used, you could probably prefix with `$EXT_BUILD_ROOT` or something. But I do think we can't assume `C` drive here.